### DISCUSSION TITLE
Checking for python-jenkins package in workstation node

### DISCUSSION THREAD

**Discussion by kvmallika** [learndevopsonline/learndevopsonline#1580] [Q&A] [answered]
Hi,

I have started practicing Jenkins as a code. In that process i checked python-jenkins package in worksation node . but i am not able to find the package . Below is the screen shot for the same .  
![image](https://github.com/learndevopsonline/learndevopsonline/assets/130459434/ee4f961c-1f6e-44f9-a4d5-af527530364e)

I have installed the ansible package again, but still i don't see the package. Please do the needful. 
Thank you

**Comment by sreedharm07** [comment, marked as answer]
hi @kvmallika 
it is available ( yum list available | grep jenkins )
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/d05e2435-b01f-42a6-b559-2eb154564924)

**Reply by kvmallika** [reply]
ok. 
Thank you
